pkg/controller/core/workload_controller_test.go

```diff
@@ -290,3 +297,53 @@ func TestSyncCheckStates(t *testing.T) {
 		})
 	}
 }
+
+func TestReconcile(t *testing.T) {
+	cases := map[string]struct {
+		workload   *kueue.Workload
+		wantError  error
+		wantEvents []utiltesting.EventRecord
+	}{
+		"admit": {
```

Comment:
Add another case where the workload is already admitted: there shouldn't be any event in that case.

Reply:
done